cleanlab/token_classification/filter.py

```diff
@@ -28,20 +30,20 @@ def find_label_issues(
 
     Returns
     ----------
-    issues: list
+    issues: List[Tuple[int, int]]
         a list containing all potential issues identified by cleanlab, such that each element is a tuple (i, j), which
         corresponds to the j'th token of the i'th sentence.
     """
     labels_flatten = [l for label in labels for l in label]
     pred_probs_flatten = np.array([pred for pred_prob in pred_probs for pred in pred_prob])
 
-    issues = find_label_issues_main(
+    issues_main = find_label_issues_main(
         labels_flatten, pred_probs_flatten, return_indices_ranked_by=return_indices_ranked_by
-    )
+    )  # type: ignore[no-untyped-call]
```

Comment:
Would omit this as well, unnecessary with our current mypy configuration. I'd be worried that we'd need to review all the `type: ignore[...]` once we switch to strict mode, to make sure that we don't have any unnecessary ones left over.

Reply:
Removed ignore-comment.